frontend/src/pages/quiz/tasks/[[...filters]].tsx

```diff
@@ -41,6 +41,9 @@ interface TasksProps {
 const Tasks: FC<TasksProps> = ({ applyingBenefits, beforeRetiring, filters, receivingBenefits }) => {
   const { t } = useTranslation('quiz/tasks')
   let router = useRouter()
+
+  if (!filters) filters = { answers: [], tags: [] }
```

Comment:
Function's argument should be immutable.

Why you shouldn't mutate parameters
https://www.seanmcp.com/articles/why-you-shouldnt-mutate-parameters/